x-pack/plugins/observability/public/components/shared/alert_search_bar/index.ts

```diff
@@ -7,3 +7,5 @@
 
 export { ObservabilityAlertSearchBar } from './alert_search_bar';
 export { ObservabilityAlertSearchbarWithUrlSync } from './alert_search_bar_with_url_sync';
+export { AlertsStatusFilter } from './components/alerts_status_filter';
```

Comment:
Where do we use this component? I cannot find the usage in this PR.

In general, I am wondering if it makes sense to re-use this component rather than copy it, especially since we want to have some improvements on this component ([ticket](https://github.com/elastic/kibana/issues/143561)) that are meaningful if this component is used in combination with the alert search bar. Because of that, we exported the whole `ObservabilityAlertSearchBar` not only `AlertsStatusFilter`

We can also continue with the current implementation and think about having different `AlertsStatusFilter` when working on the improvement ticket.